packages/core/src/Polyfill.ts

```
@@ -91,4 +92,18 @@
     }
   }
+
+  private static _registerPromiseFinally(): void {
+    // iOS Safari 10.0-10.2 and older versions do not support Promise.prototype.finally
+    // Examples: iPhone 6s and below devices without system upgrade
+    if (!Promise.prototype.finally) {
+      Logger.info("Polyfill Promise.prototype.finally");
+      Promise.prototype.finally = function <T>(this: Promise<T>, onFinally?: () => void): Promise<T> {
+        return this.then(
+          (value) => Promise.resolve(onFinally?.()).then(() => value),
+          (reason) => Promise.resolve(onFinally?.()).then(() => { throw reason; })
```

ESLint

[error] Replace `·Promise.resolve(onFinally?.()).then(()·=>·{·throw·reason;` with `⏎············Promise.resolve(onFinally?.()).then(()·=>·{⏎··············throw·reason;⏎···········`

(prettier/prettier)